docs/source/notebooks/wps_rerank_demo.ipynb

```diff
@@ -0,0 +1,203 @@
+{
```

Comment:
<p>This extra url is causing issues in <code>make</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pacificclimate/chickadee/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Oh woops!